src/components/CategoriesCard/CategoriesCard.js

```diff
@@ -0,0 +1,19 @@
+import "./CategoriesCard.css";
+
+const CategoriesCard = ({ categoryName, image }) => {
```

Comment:
you can convert the card component from your component library and then use CSS to extend its styling, that way you would also be able to sue your CSS library.

Reply:
I am using my component library card itself with minor styling updates according to my e-commerce theme.